.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+<!-- Note: for support questions, you are strongly advised to use [Stack Overflow](https://stackoverflow.com/questions/tagged/chisel). -->
```

Comment:
Can you copy this "Note" text also to the feature_request.md ?

Reply:
Yes I can

Reply:
I think it is there already.
I might be seeing it differently.

Reply:
hm, I don't see it, be sure to commit and push your changes to the correct branch?

Reply:
<img width="809" alt="Screen Shot 2022-04-18 at 9 09 05 AM" src="https://user-images.githubusercontent.com/17858596/163837350-57bd468b-bbe3-4465-acfc-96b57fac748d.png">

Reply:
I'm sorry, I was still looking at the bug_report file. But I have added it now to the feature_request.md file